styles/style.css

```diff
@@ -6,3 +6,131 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  margin: 0;
+  color: 1a1a1a;
+  font-family: "Poppins";
+}
+h3 {
+  color: #fff;
+}
+
+.navbar {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.spotify-logo {
+  width: 150px;
+  padding: 15px;
+}
+
+.navigation ul {
+  display: flex;
+  justify-content: space-between;
+}
+
+.navigation ul li {
+  list-style: none;
+  padding: 0 10px;
+  font-weight: 400;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3.5rem;
+  margin-top: 0;
+  letter-spacing: 1px;
+}
+.landing {
+  align-content: center;
+  background-image: url(../images/landing.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
There's a lot of the background image that is missing. I can recomment you to use:
background-position: center;

This will center the background image instead of showing the top part.